packages/ui/src/theme/css/component/autocomplete.scss

```diff
@@ -0,0 +1,82 @@
+%menu-space-shared {
+  padding-block-end: var(
+    --amplify-components-autocomplete-menu-space-shared-padding-block-end
+  );
+  padding-block-start: var(
+    --amplify-components-autocomplete-menu-space-shared-padding-block-start
+  );
+  padding-inline-end: var(
+    --amplify-components-autocomplete-menu-space-shared-padding-inline-end
+  );
+  padding-inline-start: var(
+    --amplify-components-autocomplete-menu-space-shared-padding-inline-start
+  );
+}
+
+.amplify-autocomplete {
+  position: relative;
+}
+
+.amplify-autocomplete__menu {
+  position: absolute;
+  z-index: 99;
+
+  width: var(--amplify-components-autocomplete-menu-width);
+  margin-top: var(--amplify-components-autocomplete-menu-margin-top);
+  background-color: var(
+    --amplify-components-autocomplete-menu-background-color
+  );
+  border-color: var(--amplify-components-autocomplete-menu-border-color);
+  border-width: var(--amplify-components-autocomplete-menu-border-width);
+  border-style: var(--amplify-components-autocomplete-menu-border-style);
+  border-radius: var(--amplify-components-autocomplete-menu-border-radius);
```

Comment:
I use a `ScrollView` for the menu to fix it. Setting first/last wont work entirely because the overflow will occur once you scroll, so basically every child :(...
![Screen Shot 2022-10-27 at 2 58 20 PM](https://user-images.githubusercontent.com/40295569/198406052-ad043d67-e7b0-405a-b5eb-351c2a2bd70d.png)